app/javascript/app/components/search/search-component.jsx

```diff
@@ -16,6 +16,12 @@ class Search extends Component {
     };
   }
 
+  componentDidMount() {
+    if (this.props.autofocus) {
+      this.inputRef.focus();
```

Comment:
This is not working on my computer. Maybe it will be better to focus when you click on the nav item

Reply:
Auto focus is only implemented in country selected page but not in country select on the nav-nested-menu.  That's because from the mobile menu the navigation will route the user to the page and it will not open the nav-nested-menu. Does it makes sense?

Reply:
It makes!